### PR TITLE
Product

### PR DESCRIPTION
**Pull request by eakansh125**

### CODE REVIEW

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image

fix positioning of badge, they are looking separate from icons

File: screens/cart.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                            </form>
+                        
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+
+
```
Comment (sri-rishi) [inline screens/cart.html]: remove extra spacing

File: screens/login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+                <div class="badge-container margin-lr">
+                    <a href="#"><img
+                            src="https://img.icons8.com/external-flatart-icons-outline-flatarticons/30/000000/external-like-instagram-flatart-icons-outline-flatarticons.png" />
+                        <span class="badge-span">6</span></a>
+                </div>
+
+                <div class="badge-container margin-lr">
+                    <a href="#"><img src="https://img.icons8.com/ios/30/000000/add-shopping-cart--v1.png" />
+                        <span class="badge-span">8</span></a>
```
Comment (sri-rishi) [inline screens/login.html]: give alt attribute for img tags

File: screens/product.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    max-width: 252px;
+    height: 100%;
+    flex-grow: 0 !important;
+}
+
+.product-container{
+    /* background-color: cadetblue; */
+    width: 100%;
+    /* margin-left: 310px; */
+    /* border: 1px solid var(--secondary-grey); */
```
Comment (sri-rishi) [inline screens/product.css]: remove comments

File: style.css [hunk 4]
```diff
@@ -302,7 +375,27 @@ body{
...
+    flex-direction: column;
+}
+
 /* Card sizes */
 .card-medium{
     width: 300px;
 }
+
+.card-small{
+    width: 250px;
```
Comment (sri-rishi) [inline style.css]: use rems instead of px

**Comment by eakansh125** [comment]
> image
> 
> fix positioning of badge, they are looking separate from icons

They have fixed position, looks a little away because of the cart icon. If I change the icon to something that has a border, you can observe the difference.

**Review by Sahil25061999 — APPROVED** [review]
Looks good just remove the comments if not necessary

**Comment by eakansh125** [comment]
> Looks good just remove the comments if not necessary

sure

**Comment by eakansh125** [comment]
Fixed badge styling and removed comments

**Review by sri-rishi — APPROVED** [review]

File: screens/cart.html (hunk 1 above)
Comment (eakansh125) [reply screens/cart.html]: done

File: style.css (hunk 4 above)
Comment (eakansh125) [reply style.css]: done

**Pull request merged**